### DISCUSSION TITLE
Who's hiring (June 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#37379] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #36738. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #37380.

**Comment by fofiu** [comment]
Monogram | **Front End Developer** | Remote (Americas Preferred) | Full-time | [Link](https://monogram.io/careers)

## Introduction
You don’t work at Monogram to remove roadblocks, you work here to pave new paths. You don’t just solve problems, you join a family of designers & developers who collaborate like clockwork, rise together, fall together, and do their life’s work here.

A front-end developer at Monogram is someone that is motivated to combine the art of design with the art of programming. An ideal front-end developer is a designer first, but a HTML and CSS wizard 🧙🏻 at the same time.

## Duties

- Collaborate on a daily basis with a team of product managers, engineers, and UI/UX designers to understand business requirements, user experience goals and pain points
- 🔥 Meticulously translate the UI/UX design wireframes to actual code mobile first
- Develop new user-facing features
- Build reusable components and libraries for future use
- Optimize application for maximum speed and scalability
- Ensure the technical feasibility of UI/UX designs

## Skills
### 🚨 Required
- Proficient in HTML5
- Solid understanding of CSS (including preprocessors like SASS)
- Tailwind CSS
- Next.js/React

### 🧁 Bonus
- TypeScript
- Knowledge of JAMstack
- Svelte
- Figma
- Three.js


### 🤔 Interested?
Send portfolio & résumé to careers@monogram.io

**Reply by aymanesarrar** [reply]
interested

**Reply by shambashib20** [reply]
Sent! Thanks for the info!

**Reply by fofiu** [reply]
Anywhere but North, Central and South America preferred. Thanks for your interest.

**Reply by fofiu** [reply]
Sure!

**Reply by codomposer** [reply]
Hello cfofiu
How are you?
I read your job posting and I am interested in this role.
This is my stack https://profile.codersrank.io/user/gitakileus
I would like to discuss more detail for this with you soon
Best regards
Lucas

**Reply by Babbili** [reply]
hello Claudio, is it still open?

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by cdaringe** [comment]
**walmart | full-stack web (various roles) | remote<sup>1</sup> | US, MX only | Full-time | $100-200k base (+ awesome benefits & RSU package) | [careers.walmart.com](https://careers.walmart.com/)**

Greetings all. We have a variety of positions open (engineers, manager, and TPM) on the walmart web platform team. Your role may include any or all of the following:

## What you'll do

- Build devtools. Help our community of teams get their work done, while keeping our solutions lightweight and effective
  - Examples of recent tasks are: integrating with our translation management service, GraphQL query optimizations/compilations, adding VSCode tasks to automate common workflows, debugging jest memory leaks, profiling next.js chunk loading, adding feature team plugins to our next.js custom server. Diverse work, with varying complexity! 
- Develop the core of 5+ next.js applications. Support the monorepo tooling for 800+ typescript libraries, and growing.
- Help manage a community.
  - We have 400+ engineers in a large typescript + next + nx monorepo
  - Design codified & documented processes and/or constraints that balance community performance with team flexibility
- Think hard, execute thoughtfully. Quality and project scalability are challenges in any project, but exacerbated when you have a large diverse community of talent and backgrounds, all racing ahead on feature work.
- Optimize processes. CI, scripts, compilers--whatever. Find the slow stuff, make it faster where you can.
- Test everything. Unit, integration, and functional tests. We're aiming for Continuous Delivery, and have the mind share and toolkit to get there. Help us get over the line!
- ...more. Being on the platform team, you also have to partner w/ our critical operational partners often. But that's a bit meta for this post 😄 

It's a very unique position. High coding, high collaboration. Beyond normal agile-ish rhythms, we meet 3x a week to structurally break down current or existential challenges. You'll enjoy it.

## What we're looking for

- required: node.js & browser excellence. if you're strong in just one, still talk to us
- required: patience. thoughtfulness. slow speaking, friendly, clear communicators
- strongly desired: typescript-fu
- desired: experience with steering a community or group


## Contact

Send me an email @ christopher.dieringer@walmart.com with `[HIRING, NEXT]: Web Platform Team - <Your Name>` as the email subject. Give me your self-evaluated skill level (junior engineer, mid-level, senior, some-thing-above :)), & ideally include your resume if available. I'll connect you with the right hiring manager based on your input! The junior roles have been generally filled, but if you compelling experience, please still reach out.

## Appendix

<sup>1</sup>: we do work remote, however, guidelines require that candidates be w/in 2hour distance from our core engineering hubs

```
// hubs

// US
Atlanta, GA
Austin, TX
Bentonville, AR
Carlsbad, CA
Dallas, TX
Hoboken, NJ
Portland, OR
Reston, VA
Seattle, WA
Sunnyvale, CA

// Non-US
Mexico City
Toronto
```

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by omure** [comment]
Omure | Senior React.js Engineer | Remote (Europe, Americas) | Full-time | [Link](http://omure.com/jobs/senior-react)

We're looking for experienced Next/Typescript devs to join our Front-end team.
There's no legacy or client work involved (we incubate startups) and hours are flexible.

Currently we offer a lot of room to grow really fast.

**Reply by codomposer** [reply]
Dear Sir,

I am writing you to apply to the position of Next Developer. My Five years of experience as a React Developer for Company l has honed my Next+Typescript skills, as well as my knowledge of mobile and responsive design.

My professional experience includes bringing the wireframes of the UX designer to life through React and Next, as well as collaborating with Back End Developers to create a stellar finished product. I think my skills and experience makes me a great fit for Omure.

Here are some of my most relevant qualifications and accomplishments:

Five years of experience as a Front End Developer

Crafted various website features using React, Next and Typescript

Developed and Implemented project scope and timelines based on the design-develop-deploy process

Collaborated with UX designers and Back End Developers and ensured coherence between all parties

Tested feature prototypes for bugs and user experience

I know how important the details are for a Front End Developer. My keen attention to detail and dedication to solving any problem at hand are what separates me from other candidates with similar qualifications. If you agree that I’d be a good fit for your organization, I would love to schedule an interview at your earliest convenience.

And one more,
These are my stack rank.
https://profile.codersrank.io/user/gitakileus
https://www.codegrepper.com/profile/dodojon-i67rx9ue4g1f

Sincerely,

akileus

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by louisguitton** [comment]
OneFootball | Frontend Engineer | DE, UK (+ remote in those countries) | Full-time | 60 - 100k EUR (for Mid level and up) | [Link](https://company.onefootball.com/career/frontend-engineer)

Hi, I'm the hiring manager. I am building a full stack web dev team for our web3 venture at OneFootball. We have a team of learners, curious people. The stack is Next.js / Typescript / Tailwind / Prisma / Postgres. We build on the Flow blockchain. We operate in a very dynamic environment with (bi-)weekly changes.

I've been at the company for 4 years and there are multiple really interesting engineering challenges that you can learn from, own and drive. We have unique football partnerships and are trying to fix the football industry.

Feel free to apply as a junior as well.

**Reply by aymanesarrar** [reply]
i'm interested in your offer

**Reply by Jahchap** [reply]
@louisguitton it seems like the link doesn't work when visited directly, and so anyone wanting to apply should please go through the following steps to access the job:
https://company.onefootball.com > Careers > Open Positions > All Openings > Frontend Engineering.

Thanks as I have applied and looking forward to getting to working with your team.

**Reply by louisguitton** [reply]
@aymanesarrar great to hear. Please use the link to apply so that I can get back to you.

**Reply by louisguitton** [reply]
@Jahchap thanks for reporting that. There seems to be an issue with the carreer website indeed. I was not aware of that and I've just passed your feedback along.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by GYusufu** [comment]
Hello all:

Herff Jones | Senior UI Architect | Remote (US) | Full-time or CTH| $130-175k base | [Link] https://www.herffjones.com/about/careers/job-openings/?gnk=job&gni=8a7883a980d90bcc0180e1abf74c193a

We're seeking someone who's an expert in Java, Hibernate, Rest APIs and also someone who is comfortable in leadership/senior positions. Fully Remote, great company culture, benefits and more. Feel free to apply using the link above or message me directly.

**Comment by saikatmitra91** [comment]
100ms | Software Engineer Web | Remote | Full-time | [Link](https://jobs.lever.co/100ms/d2e485c9-8fc6-4d31-a21c-b1e3a5315503) 

We're looking for experienced Next/Typescript/React devs to join our Front-end Self-Serve team.
There will be some work on create react app framework as well. 

- You will be part of a small team at a fast-growing engineering-first startup
- Fast-paced and performance-driven culture, career advancement and lucrative compensation, including bonuses and Esops.
- Work on a high scale, complex problems and see the real-time impact of your work.
- You will work with engineers across the globe with experience in the video

**Reply by sideeq12** [reply]
Hi saikatmitra
I'm highly interested, and readily available

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by iliassalvatore** [comment]
BEN ELLIOT | Full Stack Developer | Europe | Freelance Part-time | [Link](https://benelliot.io/about)

**We're looking for an experienced Next/React dev to join our existing Back/Front-end team**

with @PJaYOfficial and @michielvanderplas

Three.JS and WebGL parts that include an Interactive Avatar and 3D Graph have already been developed and are ready to be integrated, for the on-going development we’ve set up a Next.JS (react) framework, and Sanity Studio for back-end

**🤔 Project**
- Ben Elliot Trading Page is a way to experience Ben’s avatar and buy into his NFT token
- The goal of this project is to elevate Ben Elliot’s online presence and experience, by launching a non-fungible token (asset) Ben is allowing his art network and fans to become part of a financially driven and incentivised ecosystem
- Creating the most advanced interactive and personalised experience on the web that centralises all the data surrounding Ben Elliot on the web into one accessible place
- Art and commerce merge into a single digital commodity; $BET, a fully fledged universe where Ben Elliot’s fans can engage, interact, become shareholders and earn rewards
- This project is supported by HTC and [Viverse](https://www.vive.com/us/viverse/) as part of the [ViveArts](https://www.vivearts.com/) artist partnerships

**🚨 Required Skills**

- Full Stack Development and experience/interest in Three.JS, WebGL and 3D Animation
- Learn about real-time data streaming, displaying marketplace analytics, setting up user engagement streaks and most importantly, contribute to one of the most exciting and unique NFT projects in the Art and Crypto space to date
- By joining this project you’ll be working alongside top-tier Art Directors, Creative Web Designers, 3D Developers and become part of the Digital and Physical Art spaces, see our previous experience here: [Ilias Salvatore](https://www.linkedin.com/in/ilias-salvatore/) [PJ](https://www.linkedin.com/in/pjofficial/?originalSubdomain=nl) [Michiel van der Plas](https://www.linkedin.com/in/michielvanderplas/) and [Ben Elliot](https://www.porsche.com/dreams/global-gallery/ben-elliot)

**🚨 Duties**
- Collaborate on a daily basis with our Three.JS/WebGL developer 
- Meticulously translate the UI/UX design wireframes to actual code for web and mobile first
- Develop new user-facing features
- Build reusable components and libraries for future use
- Optimize application for maximum speed and scalability
- Ensure the technical feasibility of UI/UX designs

**📈 Timeline/budget**
- Complete Development before end-July, end to end process no longer than 3/4 weeks
- Project budget available upon request

**🤔 Interested?**
- Reach out to @iliassalvatore directly via [iliassalvatore1@gmail.com](mailto:iliassalvatore1@gmail.com)

<img width="1279" alt="BET Preview" src="https://user-images.githubusercontent.com/95893817/176457428-170d3459-3f50-41eb-b2b6-d312fd4d503a.png">

<img width="1427" alt="Screen Shot 2022-06-29 at 16 57 02" src="https://user-images.githubusercontent.com/95893817/176486017-ce934e63-4165-4612-815e-336c0aff461b.png">

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!

**Comment by VegFer** [comment]
DeFiYield | Senior Node.js Developer | Remote | Full-time | $96-132k base | [LINK](https://bit.ly/3yF9M21)

We are looking for a Senior Node.js Developer to produce scalable software solutions. You’ll be part of a cross-functional team that’s responsible for the full software development life cycle, from conception to deployment.

The experts at DEFIYIELD are specialists in DeFi yield farming and smart contract security, then have experience in investing and in developing advanced management and security toolkit.

DEFIYIELD is a blockchain security network, that allows users to generate high yields while avoiding scams and security vulnerabilities, using the safest and the most yield farming accessible ecosystem.

DEFIYIELD has developed 10+ solutions for DeFi investors, including the Advanced Automatic Smart Contract Audit Scanner - a 1-click tool that scans the smart contract code for 100+ known vulnerabilities, the Approvals Analyzer tool, World's Fist Audit, REKT Databases, and more.

Being Industry Leading Auditors, Solidity engineers performed 55+ security audits and manually collected the smart contract vulnerabilities database that became a part of the developer toolkit.

Requirements:
- Experience as a Senior Node.js Developer,
- Work experience with Node.js, Nest.js, TypeScript backend;
- Hands-on experience with relational databases, NoSQL databases;
- Experience with AWS including Lambda, SQS, CloudWatch, Cloudformation, etc.
- Experience in microservices architecture and GraphQL.
- Great attention to detail.

Responsibilities:
- Participate in product development including new features development and product maintenance,
- Develop brand new features with a distributed team and be proactive in terms of proposing new ideas,
- Develop a system from scratch.

Would be a plus:
- Experience in blockchain projects.
- Knowledge of Ethereum PRC, web3.js, the Graph.

Work conditions:
- Competitive salary corresponding to your level of knowledge, skills, and experience,
- Work with the most interesting and progressive projects in blockchain technology,
- Work Remote,
- Strong and supportive team from Canada, US, Ukraine, and Europe,
-Vacation & Paid time off,
- Learning, courses, and conferences are paid for by the company.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for July is up (https://github.com/vercel/next.js/discussions/38458) and you're more than welcome to repost this message there!
